packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -222,7 +222,7 @@ new codebuild.PipelineTestAction(this, 'IntegrationTest', {
   stage: buildStage,
   project,
   // outputArtifactName is optional - if you don't specify it,
-  // the Action will have an undefined `outputArtifact` property
+  // the Action has an undefined `outputArtifact` property
```

Comment:
I would keep the future tense here (this comment is on a construction property, when the object does not exist yet, so using the present tense feels weird here).

Reply:
Good point. I'll revert the change.